sdk/tables/Azure.Data.Tables/tests/TablesRecordedTestSanitizer.cs

```diff
@@ -2,13 +2,14 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Text.RegularExpressions;
 using Azure.Core.TestFramework;
 
 namespace Azure.Data.Tables.Tests
 {
     public class TablesRecordedTestSanitizer : RecordedTestSanitizer
     {
-        private const string SignatureQueryName = "sig";
+        private Regex SignatureRegEx = new Regex(@"([\x0026|&|?]sig=)([\w\d%]+)", RegexOptions.Compiled);
```

Comment:
```suggestion
        private Regex SignatureRegEx = new Regex(@"([^\w]sig=)([\w\d%]+)", RegexOptions.Compiled);
```
??

Reply:
I wanted to account for if the uri was already encoding unicode ampersand as well (\u0026). Perhaps that is overkilll?

Reply:
I'm fine with either option.